### DISCUSSION TITLE
easy way to delete completed pod for taskRun or pipelineRun?

### DISCUSSION THREAD

**Discussion by ec2ainun** [tektoncd/triggers#1276] [Q&A] [answered]
is there easy way to delete completed taskRun or pipelineRun that generated by TriggerTemplate?

**Comment by dibyom** [comment, marked as answer]
By easy are you talking about automatic or a single command?

Triggers adds the EventListener/Trigger info that was used to create a resource as a label so something like a `kubectl delete taskruns -l "triggers.tekton.dev/eventlistener=<EL-NAME>"` should work

For automatic cleanup, we do not have anything built in to Triggers

**Reply by ec2ainun** [reply]
thank you!
